scripts/amend_round_dates.py

```diff
@@ -39,7 +41,8 @@
     ROUND_IDS["COF_R3W1"]: cof_r3w1_config[0],
     ROUND_IDS["COF_R3W2"]: cof_r3w2_config[0],
     ROUND_IDS["COF_R3W3"]: cof_r3w3_config[0],
-    ROUND_IDS["COF_R4W1"]: cof_r4_configs[0],
```

Comment:
has this been renamed or removed? `cof_r4_configs[0]`

Reply:
it is renamed to cof_r4w1_config since I had to add a cof_r4w2_config to make it consistent

Reply:
Did we need to commit the config file that has been renamed ? `config.fund_loader_config.cof.cof_r4 `